### ISSUE TITLE
Text about decreasing stake should be removed when increasing stake:

### ISSUE THREAD

**Issue by concordium-cl** [Concordium/concordium-client#125]
**Bug Description**
Text about decreasing stake should be removed when increasing stake:
```
Note that decreasing the amount a baker is staking will lock the stake of the baker for a cooldown period before the CCD are made available.
During this period it is not possible to update the baker's stake, or stop the baker.
The current baker cooldown would last until approximately 2022-03-29 16:05:36.0646332 UTC.
```

Note that the exact same text appears correctly as warning when decreasing stake. It has no relevance when increasing stake.

**Expected Result**
No explanation of consequences of decreasing stake when increasing stake.

**Actual Result**
See above.

**Versions**
- Windows
- Concordium Client: 4.0.0-delegation-test-1
- Concordium Node: 4.0.5-delegation-test-5

**Comment by concordium-cl** [comment]
Similar issue when raising stake with `delegator configure`

<img width="847" alt="delegating3" src="https://user-images.githubusercontent.com/69570978/160786308-e064f437-1e3f-42f4-ba40-274d91c1e99e.PNG">
.

**Comment by abizjak** [comment]
Addressed in 4.0.1.
